src/app/scripts/cac/control/cac-control-itinerary-list.js

```diff
@@ -70,13 +70,19 @@ CAC.Control.ItineraryList = (function ($, Handlebars, Utils) {
 
         var source = '{{#each itineraries}}' +
                 '<div class="block block-itinerary" data-itinerary="{{this.id}}">' +
+                '<div class="trip-numbers">'+
+                '<div class="trip-duration"> {{this.durationMinutes}} min</div>'  +
+                '<div class="trip-distance"> {{this.distanceMiles}} mi</div>' +
+                '</div>' +
+                '<div class="trip-details">' +
                 '{{#each this.modes}}' +
+                '<div class="direction-icon">' +
                 ' {{modeIcon this}}' +
+                '</div>' +
                 '{{/each}}' +
-                '<span class="short-description"> via <b>{{this.via}}</b></span>' +
-                '<span class="trip-duration"> {{this.durationMinutes}} minutes,</span>'  +
-                '<span class="trip-distance"> {{this.distanceMiles}} mi.</span>' +
-                '<p><a class="itinerary" data-itinerary="{{this.id}}">View Directions</a></p>' +
+                '<span class="short-description"> via {{this.via}}</span>' +
+                '<a class="itinerary" data-itinerary="{{this.id}}">View Directions</a>' +
```

Comment:
I'm going to add a space here between 'via' and the 'View Directions' link.
![image](https://cloud.githubusercontent.com/assets/960264/7415702/a3e6949c-ef28-11e4-82ea-1767d038a3b1.png)